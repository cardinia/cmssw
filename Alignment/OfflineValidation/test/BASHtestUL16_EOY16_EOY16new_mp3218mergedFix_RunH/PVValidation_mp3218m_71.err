27-Sep-2019 22:11:57 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/B2E7C156-CE11-E911-92BC-0CC47A78A4B0.root
27-Sep-2019 22:12:15 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/B2E7C156-CE11-E911-92BC-0CC47A78A4B0.root
Begin processing the 1st record. Run 283675, Event 1268400, LumiSection 1 on stream 0 at 27-Sep-2019 22:14:30.876 CEST
Begin processing the 1001st record. Run 283676, Event 10818173, LumiSection 5 on stream 0 at 27-Sep-2019 22:23:24.820 CEST
Begin processing the 2001st record. Run 283820, Event 2694009505, LumiSection 1533 on stream 0 at 27-Sep-2019 22:23:27.510 CEST
Begin processing the 3001st record. Run 283835, Event 2557478, LumiSection 2 on stream 0 at 27-Sep-2019 22:23:31.896 CEST
Begin processing the 4001st record. Run 283835, Event 10585441, LumiSection 7 on stream 0 at 27-Sep-2019 22:23:34.113 CEST
Begin processing the 5001st record. Run 283835, Event 18769453, LumiSection 11 on stream 0 at 27-Sep-2019 22:23:37.709 CEST
Begin processing the 6001st record. Run 283885, Event 2986387098, LumiSection 1713 on stream 0 at 27-Sep-2019 22:23:42.608 CEST
Begin processing the 7001st record. Run 283885, Event 2993426386, LumiSection 1718 on stream 0 at 27-Sep-2019 22:23:45.111 CEST
Begin processing the 8001st record. Run 283885, Event 3002091460, LumiSection 1722 on stream 0 at 27-Sep-2019 22:23:49.197 CEST
Begin processing the 9001st record. Run 284006, Event 537168303, LumiSection 371 on stream 0 at 27-Sep-2019 22:23:52.351 CEST
Begin processing the 10001st record. Run 284006, Event 546451535, LumiSection 376 on stream 0 at 27-Sep-2019 22:23:55.564 CEST
Begin processing the 11001st record. Run 284006, Event 552624671, LumiSection 380 on stream 0 at 27-Sep-2019 22:23:58.914 CEST
Begin processing the 12001st record. Run 284042, Event 194249672, LumiSection 120 on stream 0 at 27-Sep-2019 22:24:04.732 CEST
Begin processing the 13001st record. Run 284042, Event 202961792, LumiSection 125 on stream 0 at 27-Sep-2019 22:24:08.352 CEST
27-Sep-2019 22:24:11 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/B2E7C156-CE11-E911-92BC-0CC47A78A4B0.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 fileAction           -s file_close                             1        1
    2 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 fileAction           PostGlobalEndRun                  
    2 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
System                  3                   3

dropped waiting message count 0
[7.571MB/7.571MB][100%][==================================================][7.571MB/s]  [7.571MB/7.571MB][100%][==================================================][7.571MB/s]  
