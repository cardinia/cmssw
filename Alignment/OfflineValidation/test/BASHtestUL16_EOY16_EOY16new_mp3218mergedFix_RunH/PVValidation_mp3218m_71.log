000 (3196555.071.000) 09/27 21:34:15 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3196555.071.000) 09/27 21:44:11 Started transferring input files
	Transferring to host: <188.185.193.10:9618?addrs=188.185.193.10-9618+[2001-1459-301-45--100-5]-9618&noUDP&sock=2832_266c_8297>
...
040 (3196555.071.000) 09/27 21:44:11 Finished transferring input files
...
001 (3196555.071.000) 09/27 21:44:11 Job executing on host: <188.185.193.10:9618?addrs=188.185.193.10-9618+[2001-1459-301-45--100-5]-9618&noUDP&sock=2461_9ea9_3>
...
006 (3196555.071.000) 09/27 21:44:20 Image size of job updated: 1808
	2  -  MemoryUsage of job (MB)
	1624  -  ResidentSetSize of job (KB)
...
006 (3196555.071.000) 09/27 21:49:21 Image size of job updated: 23412
	20  -  MemoryUsage of job (MB)
	20188  -  ResidentSetSize of job (KB)
...
006 (3196555.071.000) 09/27 21:54:22 Image size of job updated: 42892
	35  -  MemoryUsage of job (MB)
	35364  -  ResidentSetSize of job (KB)
...
006 (3196555.071.000) 09/27 21:59:22 Image size of job updated: 58548
	49  -  MemoryUsage of job (MB)
	49616  -  ResidentSetSize of job (KB)
...
006 (3196555.071.000) 09/27 22:04:23 Image size of job updated: 85388
	66  -  MemoryUsage of job (MB)
	66900  -  ResidentSetSize of job (KB)
...
006 (3196555.071.000) 09/27 22:09:24 Image size of job updated: 108148
	97  -  MemoryUsage of job (MB)
	99032  -  ResidentSetSize of job (KB)
...
006 (3196555.071.000) 09/27 22:14:24 Image size of job updated: 359888
	324  -  MemoryUsage of job (MB)
	331564  -  ResidentSetSize of job (KB)
...
006 (3196555.071.000) 09/27 22:19:24 Image size of job updated: 511040
	466  -  MemoryUsage of job (MB)
	476376  -  ResidentSetSize of job (KB)
...
006 (3196555.071.000) 09/27 22:24:25 Image size of job updated: 511040
	466  -  MemoryUsage of job (MB)
	476400  -  ResidentSetSize of job (KB)
...
040 (3196555.071.000) 09/27 22:24:35 Started transferring output files
...
040 (3196555.071.000) 09/27 22:24:35 Finished transferring output files
...
005 (3196555.071.000) 09/27 22:24:35 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:03:14, Sys 0 00:00:06  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:03:14, Sys 0 00:00:06  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	4423  -  Run Bytes Sent By Job
	1018  -  Run Bytes Received By Job
	4423  -  Total Bytes Sent By Job
	1018  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     7795        1   4011024 
	   Memory (MB)          :      466     2000      2000 
...
