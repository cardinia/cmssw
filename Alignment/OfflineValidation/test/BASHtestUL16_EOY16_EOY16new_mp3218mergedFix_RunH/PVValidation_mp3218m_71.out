Job started at  Fri Sep 27 21:44:11 CEST 2019
batch dir: /pool/condor/dir_4551 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_283675_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 7.7M -rw-r--r--. 1 tkello zh 7.6M Sep 27 22:24 PVValidation_mp3218m_283675.root -rw-r--r--. 1 tkello zh 20K Sep 27 21:44 PVValidation_mp3218m_283675_cfg.py -rw-r--r--. 1 tkello zh 3.2K Sep 27 22:24 _condor_stderr -rw-r--r--. 1 tkello zh 521 Sep 27 21:51 _condor_stdout -rwxr-xr-x. 1 tkello zh 1018 Sep 27 21:44 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 27 22:11 tkello.cc drwx------. 3 tkello zh 4.0K Sep 27 21:44 tmp drwx------. 3 tkello zh 4.0K Sep 27 21:44 var
Job ended at  Fri Sep 27 22:24:35 CEST 2019
